src/app.ts

```diff
@@ -7,8 +7,9 @@ var config = require('./controllers/config');
 var board = require('./controllers/board');
 var game = require('./controllers/game');
 var test = require('./controllers/test');
+var gameStatus = require('./controllers/gameStatus');
 
-const port = 3000;
+const port = 81;
```

Comment:
IDK how crazy we want to get, but we could possible do 

const port = process.env.PORT ? process.env.Port : 81

or whatever, and allow the docker compose to specify the port we're listening on.

Reply:
i rolled back this change, i don't want to offer my opinion yet on how to proceed here as I 
need more first hand experience with docker